### PR TITLE
Fixed issue with listDirectory

### PR DESCRIPTION
**Pull request by james-ashman**
Signed-off-by: James Ashman <james.ashman@ibm.com>

### CODE REVIEW

**Comment by james-ashman** [comment]
fixes galasa-dev/projectmanagement#706

**Comment by hobbit1983** [comment]
Have you tested this locally?

**Comment by james-ashman** [comment]
I have indeed

**Comment by james-ashman** [comment]
Just noticed that the fix should be moved to after the second if statement. Updating now.

File: galasa-managers-parent/galasa-managers-core-parent/dev.galasa.artifact.manager/src/main/java/dev/galasa/artifact/internal/BundleResourcesImpl.java [hunk 1]
```diff
@@ -422,6 +422,10 @@ public InputStream zipDirectoryContents(String resourcesDirectory, Map<String, O
     }
 
     private List<String> listDirectory(Bundle bundle, String directory, String fileExtension) {
+    	
+    	if(fileExtension != null && !fileExtension.contains(".")){
```
Comment (hobbit1983) [inline galasa-managers-parent/galasa-managers-core-parent/dev.galasa.artifact.manager/src/main/java/dev/galasa/artifact/internal/BundleResourcesImpl.java:426]: would endsWith be better?

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

File: galasa-managers-parent/galasa-managers-core-parent/dev.galasa.artifact.manager/src/main/java/dev/galasa/artifact/internal/BundleResourcesImpl.java (hunk 1 above)
Comment (james-ashman) [reply galasa-managers-parent/galasa-managers-core-parent/dev.galasa.artifact.manager/src/main/java/dev/galasa/artifact/internal/BundleResourcesImpl.java:426]: I think it should be startsWith as the file extension would be, for instance, either jar or .jar 
But yes that's a good point, I'll change it now.

**Review by hobbit1983 — APPROVED** [review]

**Pull request merged**